lib/linalg/BasisGenerator.h

```diff
@@ -263,6 +282,21 @@ class BasisGenerator
         return d_svd->getNumSamples();
     }
 
+    /**
+     * @brief Prints the summary of recommended numbers of basis vectors.
+     *
+     * @param[in] energyFractionThreshold   Energy Fraction threshold 
+     *                                      (difference of energy difference from 1.0).
```

Comment:
`difference of energy fraction from 1.0` or maybe clearer is
`energy fraction = 1.0 - energyFractionThreshold`